ios/Classes/BackgroundLocatorPlugin.m

```diff
@@ -111,6 +113,7 @@ - (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray
                                                            ARG_SPEED: @(location.speed),
                                                            ARG_SPEED_ACCURACY: @(0.0),
                                                            ARG_HEADING: @(location.course),
+                                                           ARG_TIME: @(((double) timeInSeconds) * 1000.0)  // in milliseconds since the epoch
```

Comment:
You forgot to add the same attribute for android.

Reply:
In Android, "ARG_TIME to location.time.toDouble()" in LocatorService.kt:117